internal/github/usecase/usecase.go

```diff
@@ -49,10 +49,6 @@ func (u useCase) ListRepositories(ctx context.Context, input ListRepositoriesInp
 		}
 	}
 
-	slices.SortFunc(result, func(a, b GithubRepository) int {
-		return int(b.LastUpdated.Unix() - a.LastUpdated.Unix())
-	})
```

Comment:
This sort is redundant. We already sent sort field on api and repositories come sorted